src/requests/completions.jl

```diff
@@ -168,12 +168,16 @@ function collect_completions(m::SymbolServer.ModuleStore, spartial, state::Compl
     for val in m.vals
         n, v = String(val[1]), val[2]
         (startswith(n, ".") || startswith(n, "#")) && continue
+        n_for_import = n
+        if startswith(n, "@") && endswith(n, "_str")
+            n = chop(n; head=1, tail=4) * '"'
```

Comment:
I think this breaks completion of `@xxx_str`, will fix.

Reply:
Hmm okay, so there is also a fuzzy score to check matching, not just exact matches, so I guess one would have to push two possible completions for this case.

Reply:
I now keep a vector of possible insertions per "real" name, and push them all to the candidates. Perhaps a bit convoluted for such a small feature but..